manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: KhronosGroup.VulkanSDK
+PackageVersion: 1.3.268.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --accept-licenses --default-answer --confirm-command install
+  SilentWithProgress: --accept-licenses --default-answer --confirm-command install
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: KhronosGroup.VulkanRT
+    MinimumVersion: 1.3.268.0
```

Comment:
Is this required as a dependency? Previous version seemed to have passed validation without it

Reply:
Yes otherwise the examples won't run on a system without the Runtime Installed (or by drivers) but that fails the Windows sanbox testing that was done before. See #123613